### PR TITLE
chore: remove all extendables

### LINKED ISSUE #1369: request: move all extendables to either extensions or utilities (where applicable)
**Is your feature request related to a problem? Please describe.**

Sapphire doesn't have extendables, besides, they're monkey patching and deoptimize code.

**Describe the solution you'd like**

Move the code to the equivalents. 

**Describe alternatives you've considered**

N/a.

**Additional context**

N/a.

### PR DESCRIPTION
**Pull request by kyranet**
Fixes #1369

### CODE REVIEW

File: src/commands/GameIntegration/ffxiv.ts [hunk 1]
```diff
@@ -81,10 +82,10 @@ export default class extends RichDisplayCommand {
 		const titles = language.get(LanguageKeys.Commands.GameIntegration.FFXIVCharacterFields);
 
 		const display = new UserRichDisplay(
-			new MessageEmbed()
+			new Embed()
```
Comment (favna) [inline src/commands/GameIntegration/ffxiv.ts]: This doesn't use `.splitFields` so it can just be `MessageEmbed`.

File: src/lib/discord/Embed.ts [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
-	public constructor(store: ExtendableStore, file: string[], directory: string) {
-		super(store, file, directory, { appliesTo: [MessageEmbed] });
-	}
-
-	public addBlankField(this: MessageEmbed, inline?: boolean) {
-		return this.addField(ZeroWidthSpace, ZeroWidthSpace, inline ?? false);
-	}
-
-	public splitFields(this: MessageEmbed, contentOrTitle: string | string[], rawContent?: string | string[]) {
+export class Embed extends MessageEmbed {
```
Comment (favna) [inline src/lib/discord/Embed.ts]: If we're not going to use Structures.extend for this then rename it to `SplitFieldsEmbed` to better represent what it adds

File: src/monitors/Moderation/newLineFilter.ts [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default class extends ModerationMonitor {
 
 	protected onLogMessage(message: GuildMessage, language: Language) {
 		return new MessageEmbed()
-			.splitFields(message.content)
+			.setDescription(message.content)
```
Comment (favna) [inline src/monitors/Moderation/newLineFilter.ts:48]: You're sure about these and the others? I honestly never fully understood `splitFields` but just checking.

**Review by favna — CHANGES_REQUESTED** [review]

File: src/lib/discord/Embed.ts (hunk 2 above)
Comment (kyranet) [reply src/lib/discord/Embed.ts]: We can't use Structures.extend for this.

File: src/monitors/Moderation/newLineFilter.ts (hunk 3 above)
Comment (kyranet) [reply src/monitors/Moderation/newLineFilter.ts:48]: Sets fields when they don't fit into the description.

message.content (2000 characters maximum) always fits in embed.description (2048 characters maximum).

File: src/commands/GameIntegration/ffxiv.ts (hunk 1 above)
Comment (kyranet) [reply src/commands/GameIntegration/ffxiv.ts]: Resolved in 9cbfa8fc.

File: src/lib/discord/Embed.ts (hunk 2 above)
Comment (favna) [reply src/lib/discord/Embed.ts]: So rename it to `SplitFieldsEmbed` then

**Review by favna — APPROVED** [review]

**Pull request merged**